spec/fortnox/api/validators/email_information_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'fortnox/api/validators/context'
+require 'fortnox/api/validators/email_information'
+require 'fortnox/api/models/email_information'
+
+describe Fortnox::API::Validator::EmailInformation do
+  let( :model_class ){ Fortnox::API::Model::EmailInformation }
+  let( :valid_model ){ model_class.new }
```

Comment:
Even worse then. You have a hidden dependency on the test creating the `model_class` variable. If I don't the context will fail it's tests I assume?